core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -287,6 +299,45 @@ oppia.controller('StateInteraction', [
                 stateCustomizationArgsService.displayed = {};
               };
 
+              $scope.isSaveInteractionEnabled = function() {
```

Comment:
Maybe it wasn't obvious but this is talking about the button, i.e. isSaveInteractionButtonEnabled. How do you think about that name? I don't feel too good about "Saveable".